tests/spread/integration/unzip/task.yaml

```diff
@@ -0,0 +1,15 @@
+summary: Integration tests for unzip
+
+execute: |
+  rootfs="$(install-slices base-files_bin dash_bins unzip_bins unzip_scripts zip_bins)"
+
+  chroot "${rootfs}/" unzip -hh
+  chroot "${rootfs}/" unzipsfx -hh
+  
+  echo "Hello, World" > "${rootfs}/testfile.txt"
+  chroot "${rootfs}" zip testfile.zip testfile.txt
+  chroot "${rootfs}" unzip -p testfile.zip testfile.txt > testfile_ext.txt
+  test "$( cat testfile_ext.txt )" = '$( cat "${rootfs}/testfile.txt")'
```

Comment:
This file should just have `Hello, World` in it, correct?
If so, should we just check that it's content is that, like this perhaps?
```shell
test "$(chroot "${rootfs}" unzip -p testfile.zip testfile.txt)" = "Hello, World"
```